problems/8.types.js

```diff
@@ -17,8 +17,8 @@
  * @param {number} b - the second number
  * @returns {boolean} - if a and b are both number types, return true
  */
-function bothNumbers() {
-
+function bothNumbers(a, b) {
+  return typeof a === typeof b
```

Comment:
This technically only checks that both have the same types. We would need to check if they're both numbers by doing typeof a === "number" && typeof b === "number"